codegen/render/funcs_test.go

```diff
@@ -26,6 +28,49 @@ func prepareExpected(expected string) string {
 	return expected
 }
 
+// minifyJSON is a helper function for tests to remove whitespace from JSON strings
+func minifyJSON(jsonStr string) (string, error) {
```

Comment:
json.Compact might work here https://pkg.go.dev/encoding/json#Compact

Reply:
Thanks for the pointer, I wasn't aware of that function! Just tested and it works like a charm 🎉